src/lib/journey/callbacks/text-output/text-output.stories.js

```diff
@@ -25,13 +25,13 @@ export const Base = {
   },
 };
 
-export const WithHTML = {
+export const Warning = {
```

Comment:
It may requiere a spike as im kinda new with storybook. In that case, it could be a convinient idea to perform it in all of the stories and not only for this case. thoughts?